### PR TITLE
Added NSIS installer file for Windows builds

### PR DESCRIPTION
**Pull request by samsinsane**

### CODE REVIEW

File: buildscripts/installer/windows/vaultClient.nsi [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+;Installer Sections
+
+Section "udStream" SecCore
+  SectionInstType RO
+
+  SetOutPath "$INSTDIR"
+  
+  ;Add files
+  File ..\..\..\builds\udStream${BUILD_SUFFIX}.exe
+  File ..\..\..\builds\vaultConvertCMD.exe
```
Comment (OsiJr) [inline buildscripts/installer/windows/vaultClient.nsi:102]: Probably `udStreamCMD.exe`?

Comment (samsinsane) [reply buildscripts/installer/windows/vaultClient.nsi:102]: `udStreamCMD.exe` or `udStreamConvertCMD.exe`?

Comment (OsiJr) [reply buildscripts/installer/windows/vaultClient.nsi:102]: `Convert` is better

Comment (samsinsane) [reply buildscripts/installer/windows/vaultClient.nsi]: Updated.

**Pull request merged**